### PR TITLE
Update Weaviate tutorial for 1.0.0 provider release

### PR DESCRIPTION
**Pull request by TJaniF**
The 1.0.0 version of the provider is out which changed the available operators and a few parameters. The tutorial is simpler now using automatic schema inference too :)

Note that I have not merged the code changes to main in the companion repository, I plan on merge that together with this update to avoid confusion. The updated repo is on the [dev branch](https://github.com/astronomer/airflow-weaviate-tutorial/tree/dev)

### CODE REVIEW

**Review by kentdanas — APPROVED** [review]
LGTM!

File: learn/airflow-weaviate.md [hunk 1]
```diff
@@ -72,251 +67,102 @@ The example code from this tutorial is also available on [GitHub](https://github
...
+                "X-OpenAI-Api-Key": "YOUR OPEN API KEY" # optional
+            }
+        }
+    }'
     ```
 
 :::tip
 
-See the Weaviate documentation on [environment variables](https://weaviate.io/developers/weaviate/config-refs/env-vars) and [modules](https://weaviate.io/developers/weaviate/modules/retriever-vectorizer-modules) for more information on configuring a Weaviate instance.
+See the Weaviate documentation on [environment variables](https://weaviate.io/developers/weaviate/config-refs/env-vars), [modules](https://weaviate.io/developers/weaviate/modules/retriever-vectorizer-modules) and [client instantiation](https://weaviate.io/developers/academy/zero_to_mvp/hello_weaviate/hands_on#-client-instantiation) for more information on configuring a Weaviate instance and connection.
```
Comment (lzdanski) [inline learn/airflow-weaviate.md:126]:
```suggestion
See the Weaviate documentation on [environment variables](https://weaviate.io/developers/weaviate/config-refs/env-vars), [modules](https://weaviate.io/developers/weaviate/modules/retriever-vectorizer-modules), and [client instantiation](https://weaviate.io/developers/academy/zero_to_mvp/hello_weaviate/hands_on#-client-instantiation) for more information on configuring a Weaviate instance and connection.
```

File: learn/airflow-weaviate.md [hunk 2]
```diff
@@ -72,251 +67,102 @@ The example code from this tutorial is also available on [GitHub](https://github
...
-            }
-        ]
-    }
-    ```
-
-:::tip
+```text
+1 ::: Arrival (2016) ::: sci-fi ::: A linguist works with the military to communicate with alien lifeforms after twelve mysterious spacecraft appear around the world.
+2 ::: Don't Look Up (2021) ::: drama ::: Two low-level astronomers must go on a giant media tour to warn humankind of an approaching comet that will destroy planet Earth.
+3 ::: Primer (2004) ::: sci-fi ::: Four friends/fledgling entrepreneurs, knowing that there's something bigger and more innovative than the different error-checking devices they've built, wrestle over their new invention.
```
Comment (lzdanski) [inline learn/airflow-weaviate.md:139]: The most coherent and accurate summary of Primer I've ever seen.

**Review by lzdanski — APPROVED** [review]
one typo! LGTM

File: learn/airflow-weaviate.md (hunk 2 above)
Comment (TJaniF) [reply learn/airflow-weaviate.md:139]: Haha, I wish I could take credit for that, it is the IMDB summary 😄 .

**Review by TJaniF — COMMENTED** [review]
Thank you @lzdanski ! :)

**Pull request merged**